spec/iron_bank/client_spec.rb

```diff
@@ -141,4 +141,37 @@
       expect(IronBank::Describe::Object).to have_received(:from_connection)
     end
   end
+
+  describe "DEFAULT_RETRY_OPTIONS" do
+    describe "retry_if" do
+      subject do
+        described_class::DEFAULT_RETRY_OPTIONS[:retry_if].
```

Comment:
WDYTA extracting this to a variable?
```suggestion
      let(:prok) { described_class::DEFAULT_RETRY_OPTIONS[:retry_if] }
      
      subject { prok.call(anything, exception } }
```